base/dict.jl

```diff
@@ -493,6 +546,12 @@ julia> get(d, "a", 3)
 julia> get(d, "c", 3)
 3
 ```
+
+Using `get()` is the same as using `d[x]` syntax when the key exists in the collection:
```

Comment:
I don't really see what extra information this adds that isn't in the docstring above.